File: trunk/unityClientSample/Assets/src/protocol/BaseLvRequest.cs
using ProtoBuf;
using System.Collections.Generic;
using System;

namespace protocol
{
    /// <summary>
    /// base lv 协议:22
    /// </summary>
	[Proto(value=22,description="base lv")]
	[ProtoContract]
	public class BaseLvRequest
	{
        /// <summary>
        ///  roomId
        /// </summary>
		[ProtoMember(1, IsRequired = false)]
		public string roomId;

	}
}

File: trunk/server/src/main/java/org/ngame/protocol/battle/BaseLvRequest.java
package org.ngame.protocol.battle;

import lombok.Data;
import org.ngame.protocol.annotation.Proto;

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;

/**
 *
 * @author beykery
 */
@Data
@Proto(value = 22, description = "base lv")
public class BaseLvRequest
{
	@Protobuf(description = "roomId")
	private String roomId;
}
